storage-service/src/minio.ts

```diff
@@ -29,6 +29,7 @@ interface FileWithMeta {
   meta: MetadataWithName;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Pls remove any. I think type is `Client`, more information [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/40f00cd698640f1d8077f4d52eed1b5d2b2a7838/types/minio/index.d.ts)